README.adoc

```diff
@@ -4,3 +4,4 @@ This is a simple tic-tac-toe game.
 
 All of the estimated tasks are stored in estimated.adoc file.
```

Comment:
Could provide some more information - eg title, author, how to mak it work.

Reply:
Ok sorry i've missed title and description;)

Reply:
CRC cards nicely done but it looks like UML diagram.

Reply:
README updated.

Reply:
Diagram will be updated but not today. Other stuff is just more important than that.